alerting/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java

```diff
@@ -56,15 +55,20 @@
 public class InsertedDataSubscriber {
     private static final Logger log = Logger.getLogger(InsertedDataSubscriber.class);
 
+    static final String PREFIX_AVAILABILITY = "hm_a_";
```

Comment:
These prefixes should be javadoc-ed or documented somewhere as they should be known by user.